src/main/java/uk/gov/hmcts/reform/sscs/util/MigrationDataEncoderApp.java

```diff
@@ -42,24 +37,29 @@ public static void main(String[] args) {
             log.info("Parsing migration file ({}) to generate encoded string of migration data json", MIGRATION_FILE);
 
             List<Map<String, String>> migrationData = mappingIterator.readAll();
-            migrationData.removeIf(row -> !row.get(INTERPRETER_COLUMN).trim().equals(YES.toString()));
-            migrationData.removeIf(row -> row.get(STATE_COLUMN).trim().equals(VOID_STATE.toString())
-                || row.get(STATE_COLUMN).trim().equals(DORMANT_APPEAL_STATE.toString()));
-
-            log.info("encoding data for for {} cases", migrationData.size());
+            log.info("encoding data for {} cases", migrationData.size());
 
             JSONArray migrationDataJson = new JSONArray(migrationData);
-            String encodedMigrationData = Base64.getEncoder().encodeToString(migrationDataJson.toString().getBytes());
+
+            String encodedMigrationData = compressAndB64Encode(migrationDataJson.toString());
             Path path = Paths.get(LocalDate.now().toString().replace("-", "")
-                    .concat("_" + ENCODED_STRING_FILE));
+                .concat("_" + ENCODED_STR_FILE));
             Files.write(path, encodedMigrationData.getBytes());
 
             log.info("Generated encoded string and saved it under {}", path);
 
         } catch (IOException e) {
             log.error("There was a problem encoding the migration data, migration file ({}), encoded string file ({})",
-                MIGRATION_FILE, ENCODED_STRING_FILE);
+                MIGRATION_FILE, ENCODED_STR_FILE);
             throw new RuntimeException(e);
         }
     }
+
+    public static String compressAndB64Encode(String text) throws IOException {
```

Comment:
not sure this method is covered by test

Reply:
I haven't tested this class at all, as I've considered a utility that doesn't actually get run during the migration. we run it separately beforehand.
I can test it if required